docs/src/rsh/consensus/index.md

```diff
@@ -533,6 +533,11 @@ has been called on `{!rsh} tok` yet.
 `{!rsh} Token.track(tok)`, or `{!rsh} tok.track()`, where `{!rsh} tok` is a `{!rsh} Token` value, may be used to explicitly start tracking the balance of a non-network token in an application.
 This operation is useful for tracking a token that is received from an API call or published within a container, like an `{!rsh} Array` or `{!rsh} Data` object.
 
+---
+
+`{!rsh} Token.accepted(acc, tok)`, where `{!rsh} acc` is an `{!rsh} Account` value and `{!rsh} tok` is a `{!rsh} Token` value, returns true when `{!rsh} acc` has opted in to `{!rsh} tok`, and false otherwise.
```

Comment:
We never use the term "opt in". Check out existing docs about the same concept... https://docs.reach.sh/frontend/#js_tokenAccept